### PR TITLE
Modified query to group also NULL values

### PR DESCRIPTION
**Pull request by antolinos**

### CODE REVIEW

File: ispyb-ejb/src/main/resources/queries/DataCollectionRestWsServiceBean/getViewTableQuery.sql [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
+GROUP_CONCAT(ifnull(cell_gamma,'null') SEPARATOR ', ') AS `Autoprocessing_cell_gamma`,
+GROUP_CONCAT(ifnull(anomalous,'null') SEPARATOR ', ') AS `Autoprocessing_anomalous`,
+GROUP_CONCAT(ifnull(autoProcId,'null') SEPARATOR ', ') AS `autoProcIds`,
+GROUP_CONCAT(ifnull(scalingStatisticsType,'null') SEPARATOR ', ') AS `scalingStatisticsTypes`,
+GROUP_CONCAT(ifnull(resolutionLimitHigh,'null') SEPARATOR ', ') AS `resolutionsLimitHigh`,
+GROUP_CONCAT(ifnull(resolutionLimitLow,'null') SEPARATOR ', ') AS `resolutionsLimitLow`,
+GROUP_CONCAT(ifnull(completeness,'null') SEPARATOR ', ') AS `completenessList`,
+GROUP_CONCAT(ifnull(AutoProc_spaceGroup,'null') SEPARATOR ', ') AS `AutoProc_spaceGroups`,
+GROUP_CONCAT(ifnull(meanIOverSigI,'null') SEPARATOR ', ') AS `meanIOverSigIList`,
+GROUP_CONCAT(ifnull(ccHalf,'null') SEPARATOR ', ') AS `rMerges`,
```
Comment (IvarsKarpics) [inline ispyb-ejb/src/main/resources/queries/DataCollectionRestWsServiceBean/getViewTableQuery.sql:20]: Is this a typo?

**Review by IvarsKarpics — COMMENTED** [review]
@olofsvensson  could you please have a look at my comment?

File: ispyb-ejb/src/main/resources/queries/DataCollectionRestWsServiceBean/getViewTableQuery.sql (hunk 1 above)
Comment (antolinos) [reply ispyb-ejb/src/main/resources/queries/DataCollectionRestWsServiceBean/getViewTableQuery.sql:20]: Indeed, it is

Comment (olofsvensson) [reply ispyb-ejb/src/main/resources/queries/DataCollectionRestWsServiceBean/getViewTableQuery.sql:20]: Well spotted!

**Pull request merged**